scripts/generate-apt-packages-list.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+set -e
```

Comment:
Is there a particular reason you set `errexit` but don't treat unset variables as an error?
```suggestion
set -eu
```